# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/websocket/nsWebSocketConnection.h
# Commit: 2502449c2e24
# Full Hash: 2502449c2e24993d495c886c717e7f9546cd9e17
# Author: Kershaw Chang <kershaw@mozilla.com>
# Date: 2020-08-05 21:44:54
# Regressor Bug: 1497249
# File Overlap Count: 4
# Description:
#   Bug 1497249 - P2: ipdl for nsIWebSocketConnection r=michal
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D30624
# ==============================================================================

diff -r 432c7e9dc19f -r 2502449c2e24 netwerk/protocol/websocket/nsWebSocketConnection.h
--- a/netwerk/protocol/websocket/nsWebSocketConnection.h	Mon Aug 03 10:03:08 2020 +0000
+++ b/netwerk/protocol/websocket/nsWebSocketConnection.h	Wed Aug 05 15:17:38 2020 +0000
@@ -33,6 +33,8 @@
                                  nsIAsyncInputStream* aInputStream,
                                  nsIAsyncOutputStream* aOutputStream);
 
+  nsresult EnqueueOutputData(nsTArray<uint8_t>&& aData);
+
  private:
   virtual ~nsWebSocketConnection() = default;
 
